specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ConnectedEnvironments.json

```diff
@@ -0,0 +1,449 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the connectedEnvironment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete operation completed"
+          },
+          "202": {
+            "description": "Delete operation is accepted"
+          },
+          "204": {
+            "description": "ConnectedEnvironment does not exist"
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete connected environment by name": {
+            "$ref": "./examples/ConnectedEnvironments_Delete.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ConnectedEnvironments"
+        ],
+        "summary": "Update connected Environment's properties.",
+        "description": "Patches a Managed Environment. Only patching of tags is supported currently",
+        "operationId": "ConnectedEnvironments_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the connectedEnvironment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ConnectedEnvironment"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Managed Environment": {
+            "$ref": "./examples/ConnectedEnvironments_Patch.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/connectedEnvironments/{environmentName}/checkNameAvailability": {
```

Comment:
>environmentName [](http://example.com/codeflow?start=119&length=15)

use {connectedEnvironmentName}.

The param names must be consistent

Reply:
We took a last minute PR for 2022-03-01 to make all param names consistent. [https://github.com/Azure/azure-rest-api-specs/pull/19005](https://github.com/Azure/azure-rest-api-specs/pull/19005)

We should keep it that way as inconsistent names block terraform release.